yazi-dds/src/body/delete.rs

```diff
@@ -28,7 +28,7 @@ impl<'a> From<BodyDelete<'a>> for Body<'a> {
 impl IntoLua<'_> for BodyDelete<'static> {
 	fn into_lua(self, lua: &Lua) -> mlua::Result<Value<'_>> {
 		let urls = lua.create_table_with_capacity(self.urls.len(), 0)?;
-		for (i, url) in self.urls.into_owned().into_iter().enumerate() {
```

Comment:
This seems to be a false positive. In most cases, `self.urls` will be `Cow::Owned`, so `.into_owned().into_iter()` can avoid any cloning, whereas `.iter().cloned()` will clone each element.